packages/logger/package.json

```
@@ -0,0 +1,41 @@
+{
+    "name": "@sap-ux/logger",
+    "version": "0.1.0",
+    "description": "A simple logging module",
+    "license": "SEE LICENSE IN LICENSE",
+    "private": false,
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "watch": "tsc --watch",
+        "clean": "rimraf dist coverage",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --maxWorkers=1 --colors",
+        "test-u": "jest --maxWorkers=1 --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "dependencies": {
+        "chalk": "4.1.2",
+        "lodash.clonedeep": "4.5.0",
```

```suggestion
        "lodash": "4.17.21"
```
please use `lodash` instead and `import cloneDeep from 'lodash/clonedeep';`
Makes for better treeshaking with webpack 🤷

---

Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/34fab90d6a1a69d58b44c43d0be32b66836d41df